### PR TITLE
Adding ws client tool call format 

### PR DESCRIPTION
**Pull request by OsKrg**

### CODE REVIEW

File: conversational-ai/api-reference/websocket.mdx [hunk 1]
```diff
@@ -196,6 +197,20 @@ Provides initial metadata about the conversation.
...
+**client_tool_call:**
+
+```json
+{
+  "type": "client_tool_call",
+  "client_tool_call": {
+    "tool_name": string,
+    "tool_call_id": string,
+    "parameters": any,
+    "expects_response": boolean
```
Comment (am-holt) [inline conversational-ai/api-reference/websocket.mdx:209]:
```suggestion
```
This param isn't actually here

**Review by am-holt — CHANGES_REQUESTED** [review]
Can we also add the return result type:

https://github.com/elevenlabs/xi/blob/7870c33456dbfd9455f75766797854427249b2e9/xi_backend/routes/convai/models/request_models.py#L792

**Review by louisjoecodes — APPROVED** [review]

**Comment by louisjoecodes** [comment]
LGTM!
Example for ref: {
  "client_tool_call": {
    "tool_name": "triggerName",
    "tool_call_id": "call_aqNqIDIyogksIUoXt6RMLDzb",
    "parameters": {
      "name": "Louis"
    }
  },
  "type": "client_tool_call"
}

**Review by OsKrg — COMMENTED** [review]
added response

**Pull request merged**